RNTester/js/PanResponderExample.js

```diff
@@ -10,29 +10,32 @@
 
 'use strict';
 
-var React = require('react');
-var createReactClass = require('create-react-class');
-var ReactNative = require('react-native');
-var {PanResponder, StyleSheet, View} = ReactNative;
+const React = require('react');
+const ReactNative = require('react-native');
+const {PanResponder, StyleSheet, View} = ReactNative;
 
-var CIRCLE_SIZE = 80;
+import type {PanResponderInstance} from 'PanResponder';
 
-var PanResponderExample = createReactClass({
-  displayName: 'PanResponderExample',
+type CircleStylesType = {|
+  backgroundColor: string,
+  left: number,
+  top: number,
+|};
```

Comment:
As I mentioned below, we should make all these props optional.

Reply:
👍